### PR TITLE
[DOCFIX][branch-1.5]Fix HDFS doc

### PR DESCRIPTION
**Pull request by apc999**

### CODE REVIEW

File: docs/en/Configuring-Alluxio-with-HDFS.md [hunk 1]
```diff
@@ -44,31 +46,30 @@ Worker.
...
+```
 
-### Configuring Alluxio with HDFS namenode HA mode
+### HDFS namenode HA mode
 
 If HDFS namenodes are running in HA mode, both Alluxio servers and clients should be configured
 properly in order to access HDFS.
 
-For Alluxio servers (masters and workers), copy `hdfs-site.xml` and `core-site.xml` from your hadoop
+For Alluxio servers (masters and workers), copy or make symbol-links from `hdfs-site.xml` and `core-site.xml` from your hadoop
```
Comment (gpang) [inline docs/en/Configuring-Alluxio-with-HDFS.md]: symbolic links

File: docs/en/Configuring-Alluxio-with-HDFS.md [hunk 2]
```diff
@@ -15,27 +15,29 @@ as Alluxio's under storage system.
...
+$ mvn install -P<YOUR_HADOOP_PROFILE> -DskipTests
+```
 
-{% include Configuring-Alluxio-with-HDFS/mvn-package.md %}
+Alluxio provides predefined build profiles including `hadoop-1`, `hadoop-2.2`, `hadoop-2.3` ... `hadoop-2.8` for different distributions of Hadoop. If you want to build Alluxio with a specific Hadoop release version, you can also specify the version `<YOUR_HADOOP_VERSION>` in the command. For example,
 
-* Alternatively, you can also pass the correct Hadoop version in command line when compiling with
-maven. For example, if you want Alluxio to work with Hadoop HDFS `2.6.0`:
+```bash
+$ mvn install -Phadoop-2.7 -Dhadoop.version=2.7.1 -DskipTests
```
Comment (gpang) [inline docs/en/Configuring-Alluxio-with-HDFS.md]: Do you need `-Phadoop-2.7` if you have  `-Dhadoop.version=2.7.1`?

Comment (apc999) [inline docs/en/Configuring-Alluxio-with-HDFS.md]: yes. both are needed

File: docs/en/Configuring-Alluxio-with-HDFS.md (hunk 1 above)
Comment (apc999) [inline docs/en/Configuring-Alluxio-with-HDFS.md]: done

**Comment by apc999** [comment]
@gpang PTAL

File: docs/en/Configuring-Alluxio-with-HDFS.md [hunk 3]
```diff
@@ -44,31 +46,30 @@ Worker.
 
 ## Configuring Alluxio
 
-To run Alluxio binary, we must setup configuration files. Create your configuration file with
-`bootstrapConf` command. For example, if you are running Alluxio on your local machine,
-`ALLUXIO_MASTER_HOSTNAME` should be set to `localhost`
+### Basic Configuration
 
-{% include Configuring-Alluxio-with-HDFS/bootstrapConf.md %}
+First create the configuration file from the template and set the contents.
```
Comment (calvinjia) [inline docs/en/Configuring-Alluxio-with-HDFS.md]: Remove `and set the contents`?

File: docs/en/Configuring-Alluxio-with-HDFS.md [hunk 4]
```diff
@@ -44,31 +46,30 @@ Worker.
...
+```
 
-### Configuring Alluxio with HDFS namenode HA mode
+### HDFS namenode HA mode
 
 If HDFS namenodes are running in HA mode, both Alluxio servers and clients should be configured
 properly in order to access HDFS.
 
-For Alluxio servers (masters and workers), copy `hdfs-site.xml` and `core-site.xml` from your hadoop
+For Alluxio servers (masters and workers), copy or make symbolic links from `hdfs-site.xml` and `core-site.xml` from your hadoop
```
Comment (calvinjia) [inline docs/en/Configuring-Alluxio-with-HDFS.md]: 100 character line length

File: docs/en/Configuring-Alluxio-with-HDFS.md (hunk 3 above)
Comment (apc999) [inline docs/en/Configuring-Alluxio-with-HDFS.md]: removed

File: docs/en/Configuring-Alluxio-with-HDFS.md (hunk 4 above)
Comment (apc999) [inline docs/en/Configuring-Alluxio-with-HDFS.md]: fixed

**Comment by apc999** [comment]
@calvinjia PTAL

**Pull request merged**